test/ng/directive/inputSpec.js

```diff
@@ -931,6 +946,22 @@ describe('input', function() {
     });
 
 
+    it('should be possible to override the timezone', function() {
+      var inputElm = helper.compileInput('<input type="week" ng-model="value" ng-model-options="{timezone: \'UTC\'}" />');
+
+      helper.changeInputValueTo('2013-W03');
+      expect(+$rootScope.value).toBe(Date.UTC(2013, 0, 17));
+
+      inputElm.controller('ngModel').$overrideModelOptions({timezone: '+5000'});
```

Comment:
Is this really supposed to be `+5000`?

Reply:
Revised the test and added more comments.